src/patternfly/components/Form/form.scss

```diff
@@ -76,6 +79,12 @@
       }
     }
 
+    .pf-c-form__group {
+      &.pf-m-width-100 {
+        display: block;
+      }
+    }
```

Comment:
Is this needed?

Reply:
its to undo the display: grid thats set, should i do something else ?